test/spec/marfeelTools_spec.js

```diff
@@ -41,5 +42,20 @@ describe('marfeelTools', function () {
         width: 200
       }])
     });
+
+    it('adds 1x1 to allowed sizes if 300x250 is present', function() {
```

Comment:
This is defined inside the `isBidSizeAllowed()` block. 
Shouldn't we create a new block for `getAllowedSizes`?

```suggestion
 describe('getAllowedSizes', function() {
    it('adds 1x1 to allowed sizes if 300x250 is present', function() {
```

Also, can we add a test to ensure the 1x1 is not added when there is no 300x250?